internal/designate/client/client.go

```diff
@@ -88,8 +91,10 @@ func createDesignateServiceClient() (*gophercloud.ServiceClient, error) {
 
 // ForEachZone calls handler for each zone managed by the Designate
 func (c designateClient) ForEachZone(ctx context.Context, handler func(zone *zones.Zone) error) error {
+	startTime := time.Now()
+	metrics.TotalApiCalls.Inc()
```

Comment:
I believe that this pager abstracts away one or multiple API calls, so we cannot really say here, that there was exactly one api call.